src/sonic-dhcp-server/dhcp_server/common/utils.py

```diff
@@ -93,6 +97,43 @@ def merge_intervals(intervals):
     return ret
 
 
+def validate_str_type(type, value):
+    """
+    To validate whether type is consistent with string value
+    Args:
+        type: string, value type
+        value: checked value
+    Returns:
+        True, type consistent with value
+        False, type not consistent with value
+    """
+    if not isinstance(value, str):
+        return False
+    if type not in SUPPORT_TYPE:
+        syslog.syslog(syslog.LOG_ERR, "Unsupported validate type: {}".format(type))
```

Comment:
I suggest not print log in util method.
Print log in where you invoke it and got an unexpected result

Reply:
Fixed